### DISCUSSION TITLE
[Feature Request] Move issue to In-progress when PR linked

### DISCUSSION THREAD

**Discussion by gpotter2** [community/community#22359] [New to GitHub]
<p>Hi,</p>
<p>It would be great if there was an automation option in Projects that allowed to move an issue from TODO to In-progress whenever a PR references the issue (the “may be fixed by” that github displays).</p>
<p>I’ve seen this idea mentionned here: <a href="https://github.community/t/github-projects-issues-and-pull-requests/1353/11">Github community issue 1353</a> but I’m not sure that it got noticed.</p>
<p>Thanks for making Github so great !</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi <a class="mention" href="/gpotter2">@gpotter2</a> thanks for sharing your thoughts here about this feature.</p>
<p>We’re always working to improve GitHub and the GitHub Support Community, and we consider every suggestion we receive.<br>
Would you mind <a href="https://support.github.com/contact/feedback">submitting this through our official product feedback form</a> so that our product team can track your request?</p>

**Comment by gpotter2** [comment]
<p>I just did, Thanks for the answer.</p>

**Comment by Rick-Anderson** [comment]
<p>Where is the suggestion. I’d like to upvote it.</p>
